### PR TITLE
[IMPROVE] Emoji input preview

### PR DESCRIPTION
**Pull request by RahulNavneeth**
# Preview emojis in text-area

## description
Adds the ability to preview emojies in the text-area

## Video/Screenshots

BEFORE :

https://user-images.githubusercontent.com/50259284/224507546-4d90c771-d913-4413-8434-993b537e2064.mp4

AFTER:

https://user-images.githubusercontent.com/50259284/224507549-42ac22ca-7483-4d68-84a0-b0f5af162f2b.mp4

### CODE REVIEW

**Comment by RahulNavneeth** [comment]
Hello @sidmohanty11 , Can you review this pull request? Thank you very much! 😄

File: src/components/ChatInput/ChatInput.js [hunk 1]
```diff
@@ -163,6 +164,17 @@ const ChatInput = () => {
     }
   };
 
+  const parseEmoji = (text) => {
```
Comment (sidmohanty11) [inline src/components/ChatInput/ChatInput.js:167]: great work, please take this function inside - `lib/emoji.js` (create one)

**Review by sidmohanty11 — CHANGES_REQUESTED** [review]
Thanks @RahulNavneeth !

File: src/components/ChatInput/ChatInput.js (hunk 1 above)
Comment (RahulNavneeth) [reply src/components/ChatInput/ChatInput.js]: Done sir 👍

**Review by sidmohanty11 — APPROVED** [review]

**Pull request merged**